Tests/BotTests/XCTestManifests.swift

```diff
@@ -1,6 +1,21 @@
 #if !canImport(ObjectiveC)
 import XCTest
 
+extension DispatchServiceTests {
+    // DO NOT MODIFY: This is autogenerated, use:
+    //   `swift test --generate-linuxmain`
+    // to regenerate.
+    static let __allTests__DispatchServiceTests = [
```

Comment:
btw didn't they release automatic test discovery on linux? or it's just in master?

Reply:
Was not aware of that, if that's the case that would be wonderful, given how many times we forgot to update those files!

Reply:
some info here https://forums.swift.org/t/test-discovery-on-linux/26203 it's actually SPM feature

Reply:
Cool, thanks for the link; that looks promising!
Not sure what to conclude from that thread regarding if it's already released (or still only on master though), but probably not yet; but clearly something to keep a look at 👍